cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -55,14 +55,18 @@ <h3 class="h4">
 
    Creates related metadata list markup when given:
 
-   list: A list of dictionaries containing links.
+   list:              A list of dictionaries containing links.
+
+   is_related_topics: Boolean indicating whether the list is a
+                      Related Topics list. Defaults to false.
 
    ========================================================================== #}
 
-{% macro _list(list) %}
+{% macro _list(list, is_related_topics=false) %}
 <ul class="list
            list__unstyled
-           list__links">
+           list__links
+           {{ 'm-related-metadata_topics' if is_related_topics else '' }} ">
     {% for link in list.links %}
     <li class="list_item">
         <a href="{{ link.url }}" class="list_link">
```

Comment:
The tags don't have link data. Should this just be the tag text here? This might merit just copying the markup over to the _topics() macro all together since {% for link in list.links %} doesn't really work for it either since the tags is just a list of strings.